src/cli/sandbox.cpp

```diff
@@ -63,6 +67,56 @@ bool Sandbox::init()
       }
 
    return (::cap_enter() == 0);
+#elif defined(BOTAN_TARGET_OS_HAS_SETPPRIV)
+   priv_set_t *ps;
+
+   if ((ps = ::priv_allocset()) == nullptr)
+      {
+      return false;
+      }
+
+   ::priv_basicset(ps);
+
+   if (::priv_delset(ps, PRIV_PROC_FORK) == -1)
```

Comment:
These and following could be replaced by a loop, something like:

const auto privs[] = { PRIV_PROC_FORK, PRIV_PROC_EXEC, ... }
for(auto priv : privs) {
  priv_delset(ps, priv);
}